sdk/cosmos/azure-cosmos/tests/test_fault_injection_transport.py

```diff
@@ -62,8 +63,13 @@ def teardown_class(cls):
             logger.warning("Exception trying to delete database {}. {}".format(created_database.id, containerDeleteError))
 
     @staticmethod
-    def setup_method_with_custom_transport(custom_transport: RequestsTransport, default_endpoint=host, **kwargs):
-        client = CosmosClient(default_endpoint, master_key, consistency_level="Session",
+    def setup_method_with_custom_transport(custom_transport: RequestsTransport,
+                                           default_endpoint=host,
+                                           use_multiple_write_locations=False,
+                                           **kwargs):
+        connection_policy = ConnectionPolicy()
```

Comment:
just curious, is the connection policy used here necessary for anything in the test?